### PR TITLE
Enhancement to "test_pv_scale_and_respin_ceph_pods" test with additional resource types

### PR DESCRIPTION
**Pull request by pintojoy**
Enhancement to "test_pv_scale_and_respin_ceph_pods". Apart from existing four resource type (mgr, mon, osd and mds) in ceph resource types, additional resource types are added (rbdplugin_provisioner, cephfsplugin_provisioner, cephfsplugin, rbdplugin and operator)

Enhancements are part of "Automate scale tests with CSI pod re-spin" test

### CODE REVIEW

**Review by Avilir — DISMISSED** [review]
LGTM

File: tests/e2e/scale/test_pv_scale_and_respin_ceph_pods.py [hunk 1]
```diff
@@ -132,6 +133,15 @@ def cleanup(self):
...
         pytest.param(*["osd"], marks=[pytest.mark.polarion_id("OCS-765")]),
         pytest.param(*["mds"], marks=[pytest.mark.polarion_id("OCS-613")]),
+        pytest.param(
+            *["cephfsplugin_provisioner"], marks=[pytest.mark.polarion_id("OCS-2641")]
+        ),
+        pytest.param(
+            *["rbdplugin_provisioner"], marks=[pytest.mark.polarion_id("OCS-2639")]
+        ),
+        pytest.param(*["rbdplugin"], marks=[pytest.mark.polarion_id("OCS-2643")]),
+        pytest.param(*["operator"], marks=[pytest.mark.polarion_id("OCS-2640")]),
```
Comment (keesturam) [inline tests/e2e/scale/test_pv_scale_and_respin_ceph_pods.py:143]: operator restart might not be relevant here.

Comment (pintojoy) [reply tests/e2e/scale/test_pv_scale_and_respin_ceph_pods.py]: Removed operator resource.

**Review by keesturam — APPROVED** [review]
LGTM

**Review by ramkiperiy — APPROVED** [review]
LGTM

**Pull request merged**